### DISCUSSION TITLE
.Net MAUI Hello World Application with VS2019 Preview

### DISCUSSION THREAD

**Discussion by intzarali** [dotnet/maui#1132] [General]
Hi,

I am try to create Hello World application of .Net MAUI in VS2019 preview.
But getting various errors.

Failed to restore D:\UNO_PLATFORM\HelloMaui\HelloMaui\HelloMaui.csproj (in 4.07 sec).
Failed to restore D:\UNO_PLATFORM\HelloMaui\HelloMaui.WinUI3\HelloMaui.WinUI3.csproj (in 5.1 sec).
NuGet package restore failed. Please see Error List window for detailed warnings and errors.

and how to install Maui packages.

using Microsoft.Maui;
using Microsoft.Maui.Controls;

please guide me on this. or suggest any article to develop the hello world MAUI app

Thank in Advance,
Intzar

**Comment by lbcsy** [comment]
These package seems not available in NuGet gallery, but they are in Maui samples: https://github.com/dotnet/maui-samples.
What you can do is clone that repo and copy those package out and put them into a local NuGet folder: C:\Program Files (x86)\Microsoft SDKs\NuGetPackages\ and they will be restored into your projects.

**Comment by davidortinau** [comment]
With Preview 4 you need our public NuGet feed hosted on Azure. Follow these steps to add it. https://github.com/dotnet/maui/wiki/Getting-Started
